docs/calicoctl/pool.md

```diff
@@ -0,0 +1,145 @@
+
+# User guide for 'calicoctl pool' commands
+> NOTE: The `calicoctl pool` configuration commands should NOT be used when 
```

Comment:
"NOTE: ONLY the `calicoctl pool show` command is safe to use when running Calico with the Docker libnetwork driver. ..."